db/migrate/20191126181009_seed_test_results.rb

```diff
@@ -1,33 +1,35 @@
 class SeedTestResults < ActiveRecord::Migration[5.2]
   def up
     migrated = 0
-    ProfileHost.find_in_batches(batch_size: 200) do |profile_host_group|
+    ProfileHost.find_in_batches(batch_size: 30) do |profile_host_group|
       test_results = []
       rule_results_set = []
       profile_host_group.each do |profile_host|
-        host = profile_host.host
-        profile = profile_host.profile
+        host = profile_host.host_id
+        profile = profile_host.profile_id
```

Comment:
Let's rename these variables to indicate they're just IDs

```suggestion
        host_id = profile_host.host_id
        profile_id = profile_host.profile_id
```